dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -247,20 +280,34 @@ private List<WebServiceFeature> getAllFeatures(WebServiceFeature[] features) {
 
     private JaxWsClientEndpointImpl getJaxwsEndpoint(QName portName, AbstractServiceFactoryBean sf,
                                       WebServiceFeature...features) {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         Service service = sf.getService();
         EndpointInfo ei;
         if (portName == null) {
             ei = service.getServiceInfos().get(0).getEndpoints().iterator().next();
+            if (isFinestEnabled) {
+                LOG.finest("Jaxws endpoint that is obtained from service > ServiceInfo(0) since portname is null: " + ei);  // Liberty Change issue #26529
+            }
         } else {
             ei = service.getEndpointInfo(portName);
+            if (isFinestEnabled) {
+                LOG.finest("Jaxws endpoint that is obtained from service using portname: " + ei);  // Liberty Change issue #26529
+            }
             if (ei == null) {
                 PortInfoImpl portInfo = getPortInfo(portName);
+                if (isFinestEnabled) {
+                    LOG.finest("PortInfoImpl that is obtained from ServiceInpl.portInfos with key portname: " + portInfo);  // Liberty Change issue #26529
+                }
                 if (null != portInfo) {
                     try {
                         ei = createEndpointInfo(sf, portName, portInfo);
                     } catch (BusException e) {
                         throw new WebServiceException(e);
                     }
+                    if (isFinestEnabled) {
+                        LOG.finest("Endpoint was null, it's created instead of obtained: " + ei);  // Liberty Change issue #26529
```

Comment:
I would change the trace message to LOG.finest("Endpoint was null, it's created using portInfo: " + ei);